web-client/src/client/block.rs

```diff
@@ -22,6 +22,8 @@ pub struct PlainBlockCommonFields {
     /// The timestamp of the block. It follows the Unix time and has millisecond precision.
     pub timestamp: u64,
 
+    /// The network that this block is valid for.
+    pub network: &'static str,
```

Comment:
@hrxi You are using `&'static str` instead of the `String` type I have been using everywhere else. Is there an advantage to using either?

Reply:
`String` is the more general-purpose type. `&'static str` only works for strings already known at compile time. Since we know all the possible networks, `&'static str` makes it a bit more efficient since it doesn't need an allocation.